arnoldPlugins/gaffer.mtd

```diff
@@ -1463,6 +1467,10 @@
 	[attr filename]
 		widget STRING "filename"
 		gaffer.layout.index INT 0
+		gaffer.path.valid BOOL true
+		gaffer.path.leaf BOOL true
+		gaffer.path.bookmarks STRING "texture"
+		gaffer.fileSystemPath.extensions STRING "tx"
```

Comment:
I don't know how common it is to use Arnold's automatic .tx conversion - would it be worth registering the `aiImage` extensions to `GafferImage.ImageReader.supportedExtensions()` so more image formats are available without having to click off the filter?

Reply:
I did wonder about doing that, and wasn't sure what to do. We've gone out of our way to [default auto-tx-generation off](https://github.com/GafferHQ/gaffer/commit/50f8950ccd5725b6516709ce16f20bebd8b4bc7a), because it's not compatible with render-farm usage. And I think maybe gently encouraging people away from using non-mip-mapped formats is a pretty reasonable thing to do. So I may be wrong, but on balance I think it's reasonable to only show the ideal format by default, and have people toggle a checkbox to get at the others...

Reply:
That sounds reasonable to me, clear to merge from me.